src/app/pages/applications/chart-releases/chart-releases.component.ts

```diff
@@ -65,21 +62,45 @@ export class ChartReleasesComponent implements OnInit {
     private sysGeneralService: SystemGeneralService) { }
 
   ngOnInit(): void {
+    this.getCatalogApps();
     this.refreshChartReleases();
-    this.refreshForms();
+    this.utils = new CommonUtils();
+
     this.refreshTable = this.modalService.refreshTable$.subscribe(() => {
       this.refreshChartReleases();
-    })
-    this.refreshForm = this.modalService.refreshForm$.subscribe(() => {
-      this.refreshForms();
     });
   }
 
-  refreshForms() {
-    this.chartReleaseForm = new ChartReleaseEditComponent(this.mdDialog,this.dialogService,this.modalService,this.appService);
-    this.plexForm = new PlexFormComponent(this.mdDialog,this.dialogService,this.modalService,this.sysGeneralService,this.appService);
-    this.nextCloudForm = new NextCloudFormComponent(this.mdDialog,this.dialogService,this.modalService,this.appService);
-    this.minioForm = new MinioFormComponent(this.mdDialog,this.dialogService,this.modalService);
+  getCatalogApps() {
+    this.appService.getAllCatalogItems().subscribe(res => {
+      res.forEach(temp => {
```

Comment:
Maybe a different name than "temp"?
Something that more clearly describes what "temp"is, in this case it seems to be CatalogItems, so maybe "items" or something like it?

Reply:
Solved, looks good! :)